public/electron.js

```diff
@@ -94,6 +142,32 @@ if (process.platform === 'darwin') {
 // Some APIs can only be used after this event occurs.
 app.on('ready', createWindow)
 
+function startSystray() {
+  tray = new Tray(path.join(__dirname, trayIcon));
+  const contextMenu = Menu.buildFromTemplate([
+    {
+      label: 'Hide wallet/Show wallet', 
```

Comment:
We should set this label as 'Hide Wallet' when `mainWindow.isVisible()` is `true` and 'Show Wallet', otherwise